### ISSUE TITLE
is this right? 

### ISSUE THREAD

**Issue by mhartington** [yosiat/oceanic-next-vim#5]
So I followed the instructions, but the colors don't appear right.

![screen shot 2015-06-05 at 5 44 00 pm](https://cloud.githubusercontent.com/assets/2835826/8017436/92dcfbb6-0baa-11e5-99b0-cc96413a90d0.png)

This is vim in iterm2.

**Comment by mhartington** [comment]
Just a comparison, it seem things work fine on macvim, but not terminal vim.
![screen shot 2015-06-06 at 9 16 45 am](https://cloud.githubusercontent.com/assets/2835826/8020429/e5e4935a-0c2c-11e5-9326-be2babbc6900.png)

**Comment by yosiat** [comment]
It looks weird.. It happens to me as well.. 
I have no idea how to fix this =X

**Comment by mhartington** [comment]
Hmm, alright, I'll take a look and if I can get something together, I'll make a PR

**Comment by yosiat** [comment]
Thanks, I will try to check something.. and let you know.

**Comment by mhartington** [comment]
Cool, and if it's not possible, we could always checkout this project 
https://github.com/chriskempson/base16-builder

It should allow us to make a vim color scheme that will work with gui-vim and term-vim (plus a lot of other places)

**Comment by bendyorke** [comment]
Did you ever find a solution to this?  I really like this theme but am running into the same problem.

**Comment by mhartington** [comment]
I don't think there is an solution to this. It has to do with the color space and term-vim has vs macvim. 
I have sense replaced this with a fork of the base16-shell and base16-vim combo to get oceanic-next to work
![screen shot 2015-07-08 at 3 39 31 pm](https://cloud.githubusercontent.com/assets/2835826/8580252/930f8f80-2587-11e5-95ba-eee9693eb7b1.png)

https://github.com/mhartington/base16-vim
https://github.com/mhartington/base16-shell

Closing as this is a limitation of terminal vim, not an issue with the theme.

**Comment by gabrielflorit** [comment]
@mhartington  @yosiat @bendyorke I finally got this working on iTerm 3 beta and latest Vim:

<img width="700" alt="tbyvpth" src="https://cloud.githubusercontent.com/assets/370976/9780465/11478ee2-5757-11e5-9262-080ec774870f.png">

Instructions to follow.

**Comment by gabrielflorit** [comment]
**NEW instructions**

To get the Vim theme:
- It's part of [base16-builder](https://github.com/chriskempson/bas16-builder). 

**OLD instructions**

To get the Vim theme:
- Install [chriskempson/base16-vim](https://github.com/chriskempson/base16-vim)
- Grab my fork of base16-builder [gabrielflorit/base16-builder](https://github.com/gabrielflorit/base16-builder)
- Run `./base16` and find `oceanicnext.vim` in the `output` directory
- Enjoy!

**Comment by mhartington** [comment]
@gabrielflorit were you running the bash16 shell script as well? That was the only way I could got it to work. I have since gone and made my own variation of oceanic next for vim 
https://github.com/mhartington/oceanic-next

**Comment by gabrielflorit** [comment]
No, I set both vim and iterm to their respective base16-oceanicnext color schemes.

**Comment by yosiat** [comment]
Hi @gabrielflorit , can you submit a pull request for those fixes?

**Comment by gabrielflorit** [comment]
@yosiat It was recently folded into [base16-builder](https://github.com/chriskempson/base16-builder). I'd encourage everybody to check that out instead.

**Comment by olalonde** [comment]
Having same problem here:

<img width="896" alt="screenshot 2016-03-07 12 37 38" src="https://cloud.githubusercontent.com/assets/416585/13582561/d8631db0-e461-11e5-869f-933fc44553c9.png">

and I'm not sure I understand what the fix is...

**Comment by gabrielflorit** [comment]
@olalonde are you using [base16-builder](https://github.com/chriskempson/base16-builder)?

**Comment by olalonde** [comment]
Yeah though it's not really clear to me what its purpose is. Why isn't the theme pre built? That's what I did so far but not getting anything in output/:

<img width="435" alt="screenshot 2016-03-07 12 56 17" src="https://cloud.githubusercontent.com/assets/416585/13583005/1bda75fa-e464-11e5-935c-7bd0f1321e6e.png">

**Comment by mhartington** [comment]
Just adding, that I no longer use this theme, but have gone and built my own version of oceanic-next.

https://github.com/mhartington/oceanic-next

Feel free to try that out as well. All setup for vim and neovim

**Comment by olalonde** [comment]
@mhartington thanks, will give this a try

**Comment by yosiat** [comment]
Hi guys!

I am sorry for not commenting and not being active on this,
I don't have the time to maintain and this project and there is a great alternative created by @mhartington  - https://github.com/mhartington/oceanic-next

**Comment by olalonde** [comment]
Thank, ended up using @mhartington scheme.
